lib/matplotlib/stackplot.py

```diff
@@ -23,14 +23,14 @@ def stackplot(axes, x, *args,
 
     Parameters
     ----------
-    x : (N,) array-like
+    x : (N, ) array-like
 
     y : (M, N) array-like
-        The data is assumed to be unstacked. Each of the following
+        The data can be either stacked or unstacked. Each of the following
         calls is legal::
+            stackplot(x, y) # where y has shape (M, N) e.g. y = [y1, y2, y3, y4]
+            stackplot(x, y1, y2, y3, y4) # where y1, y2, y3, y4 have length N
 
```

Comment:
```suggestion
            stackplot(x, y)  # where y has shape (M, N) e.g. y = [y1, y2, y3, y4]
            stackplot(x, y1, y2, y3, y4)  # where y1, y2, y3, y4 have length N

```

Use two spaces before a comment as is convention in python code.

Leave the surrounding empty line as is; see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

> like all paragraphs, separated from the surrounding ones by blank lines